libs/blocks/utils/utils.js

```diff
@@ -26,4 +28,12 @@ const fetchWithTimeout = async (resource, options = {}) => {
   return response;
 };
 
-export { getLocalStorage, setLocalStorage, fetchWithTimeout };
+const updateLinkWithConfiguredLangRoot = (link) => {
+  const langRoot = getMetadata('lang-root');
+  if (!langRoot) return link;
+  const url = new URL(link);
```

Comment:
if `link` is an invalid URL it will throw a TypeError

Reply:
good catch, addressed in https://github.com/adobecom/milo/pull/1101/commits/e2149877f76d990b95a22a56b442badd24f5c7b6